packages/subgraph/test/helpers/helpers.ts

```diff
@@ -455,7 +456,11 @@ export const clipDepositNumber = (deposit: BigNumber, roundingDown = false) => {
     const rounding = roundingDown
         ? 0
         : deposit.and(toBN(0xffffffff)).isZero()
-        ? 0
-        : 1;
+            ? 0
+            : 1;
     return deposit.shr(32).add(toBN(rounding)).shl(32);
 };
+
+export const getOrder = (blockNumber?: number, logIndex?: number) => {
+    return blockNumber! * 10000 + logIndex!
```

Comment:
I think it will be good to add a note that this number is defined as `ORDER_MULTIPLIER` in `packages/subgraph/src/utils.ts`